signer/src/config/default.toml

```diff
@@ -1,19 +1,124 @@
+# TODO: Add documentation for all configuration parameters.
```

Comment:
Can we change this to:
```toml
# TODO(429): Add documentation for all configuration parameters.
```
We typically add the ticket associated with a TODO so that they do not get lost.